### PR TITLE
Persist tooltip transform into localStorage

### PR DESCRIPTION
**Pull request by julianrubisch**

### CODE REVIEW

File: src/elements/tooltip_element.js [hunk 1]
```diff
@@ -5,6 +5,25 @@ export default class TooltipElement extends HTMLElement {
...
+
+  disconnectedCallback () {
+    if (this.id !== 'undefined' && this.id !== '') {
+      localStorage.setItem(
+        `turbo-boost-devtools-${location.href}-${this.id}-transform`,
+        this.style.transform
+      )
+    }
+  }
+
```
Comment (hopsoft) [inline src/elements/tooltip_element.js:26]: Is `transform` really the only CSS property that needs to be persisted?

File: src/devtool.js [hunk 2]
```diff
@@ -202,7 +202,8 @@ export default class Devtool {
     `
     const tooltip = appendTooltip(title, subtitle, content, {
       backgroundColor: 'lightyellow',
-      color: 'chocolate'
+      color: 'chocolate',
+      id: `${this.delegate.id}-rendering`
```
Comment (hopsoft) [inline src/devtool.js]: I might move `id` to the top of the "list" just to help denote its importance.

File: src/devtool.js [hunk 3]
```diff
@@ -240,7 +241,8 @@ export default class Devtool {
       {
         backgroundColor: 'lightcyan',
         color: 'darkcyan',
-        position: 'bottom'
+        position: 'bottom',
+        id: `${this.delegate.id}-target`
```
Comment (hopsoft) [inline src/devtool.js]: See related position comments for `id`.

**Review by hopsoft — APPROVED** [review]

File: src/elements/tooltip_element.js (hunk 1 above)
Comment (julianrubisch) [reply src/elements/tooltip_element.js:26]: it does seem so. if I find another one, I can add it...

**Pull request merged**